HW03/Tatsiana_Zenkovich/app.rb

```diff
@@ -0,0 +1,80 @@
+class Student
+  attr_reader :name, :surname
+  def initialize(name, surname)
+    @name = name
+    @surname = surname
+    @homeworks = []
+    @mentor = []
+  end
+
+  def submit_homework!(homework_data)
+    @homeworks << homework_data
+    # puts 'homework submitted'
+    notify_mentor
+  end
+
+  def notify_mentor
+    @mentor.each { |mentor| mentor.update(self) }
+    # puts 'mentor notified'
+  end
+
+  def homeworks
+    puts @homeworks
+  end
```

Comment:
You do not need this method if you add homeworks to attr_???
Or just return homework
```suggestion
  def homeworks
    @homeworks
  end
```

Reply:
corrected, as you suggested